lib/kuzzle/kuzzle.js

```diff
@@ -57,6 +59,9 @@ class Kuzzle extends KuzzleEventEmitter {
       config.plugins.common.maxConcurrentPipes,
       config.plugins.common.pipesBufferSize);
 
+    // Node unique identifier
+    this.id = randomBytes(16).toString('hex');
```

Comment:
Ok, why not. :+1:

Reply:
Also, this could be used to identify Kuzzle node connection to Redis.  
If it's not too much work you could add it in this PR, otherwise we can do it in a separate PR.

See https://github.com/kuzzleio/kuzzle/issues/1655

Reply:
already done ;)

Reply:
Good thinking. And done, all clients are now identified, even the cluster. Format is: "adapter name/node id"

```
127.0.0.1:6379> client list

id=105 addr=192.168.112.6:43298 fd=15 name=cluster_manager/general-sloppy-run-6762 age=253 idle=2 flags=N db=0 sub=0 psub=0 multi=-1 qbuf=0 qbuf-free=0 obl=0 oll=0 omem=0 events=r cmd=smembers
id=100 addr=192.168.112.4:57952 fd=10 name=public_adapter/military-loud-competition-5560 age=254 idle=253 flags=N db=5 sub=0 psub=0 multi=-1 qbuf=0 qbuf-free=0 obl=0 oll=0 omem=0 events=r cmd=info
id=101 addr=192.168.112.4:57954 fd=11 name=internal_adapter/military-loud-competition-5560 age=254 idle=3 flags=N db=0 sub=0 psub=0 multi=-1 qbuf=0 qbuf-free=0 obl=0 oll=0 omem=0 events=r cmd=set
id=102 addr=192.168.112.6:43254 fd=12 name=public_adapter/general-sloppy-run-6762 age=254 idle=253 flags=N db=5 sub=0 psub=0 multi=-1 qbuf=0 qbuf-free=0 obl=0 oll=0 omem=0 events=r cmd=info
id=107 addr=127.0.0.1:54748 fd=17 name= age=28 idle=0 flags=N db=0 sub=0 psub=0 multi=-1 qbuf=26 qbuf-free=32742 obl=0 oll=0 omem=0 events=r cmd=client
id=103 addr=192.168.112.6:43256 fd=13 name=internal_adapter/general-sloppy-run-6762 age=254 idle=3 flags=N db=0 sub=0 psub=0 multi=-1 qbuf=0 qbuf-free=0 obl=0 oll=0 omem=0 events=r cmd=set
id=106 addr=192.168.112.4:58012 fd=16 name=cluster_manager/military-loud-competition-5560 age=253 idle=2 flags=N db=0 sub=0 psub=0 multi=-1 qbuf=0 qbuf-free=0 obl=0 oll=0 omem=0 events=r cmd=smembers
id=104 addr=192.168.112.5:34274 fd=14 name=cluster_manager/wry-soggy-committee-4186 age=253 idle=3 flags=N db=0 sub=0 psub=0 multi=-1 qbuf=0 qbuf-free=0 obl=0 oll=0 omem=0 events=r cmd=smembers
id=98 addr=192.168.112.5:34220 fd=8 name=internal_adapter/wry-soggy-committee-4186 age=254 idle=3 flags=N db=0 sub=0 psub=0 multi=-1 qbuf=0 qbuf-free=0 obl=0 oll=0 omem=0 events=r cmd=set
id=99 addr=192.168.112.5:34222 fd=9 name=public_adapter/wry-soggy-committee-4186 age=254 idle=253 flags=N db=5 sub=0 psub=0 multi=-1 qbuf=0 qbuf-free=0 obl=0 oll=0 omem=0 events=r cmd=info
```